src/VisualStudio/VisualStudioInteractiveComponents/source.extension.vsixmanifest

```diff
@@ -4,13 +4,21 @@
     <Identity Id="500fff63-afcf-4195-8db4-3fa8a5180e79" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Interactive Components</DisplayName>
     <Description>Interactive Roslyn components for Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\InteractiveComponents</ShortcutPath>
   </Metadata>
   <Installation Experimental="true">
     <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0,]" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.VisualBasic.InteractiveEditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.CSharp.InteractiveEditorFeatures.dll" />
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.InteractiveEditorFeatures.dll" />
```

Comment:
This VSIX also seems to be pulling in bunch of the same dlls as the Language Services VSIX (MS.CA, MS.CA.CSharp, workspaces, features etc.). This may be fine for now but I think we will need to avoid packaging the same binary in multiple VSIXes eventually.